src/Navigation/Notifications/Repository/Notify.php

```diff
@@ -677,19 +677,7 @@ public function NotifyOnDesktop(Entity\Notification $Notification, string $type
 
 		$notify_type = $this->pConfig->get($Notification->uid, 'system', 'notify_type', 3 | 72 | 4 | 16 | 32);
```

Comment:
Please use the constant in this statement as well.

Reply:
I did it now slightly different.